ledger/participant-state/kvutils/BUILD.bazel

```diff
@@ -201,11 +201,11 @@ client_server_build(
         "--contract-id-seeding=testing-weak",
         "--participant participant-id=%s,port=%d" % (REFERENCE_LEDGER_EXPORT_NAME, REFERENCE_LEDGER_EXPORT_PORT),
     ],
-) if not is_windows else None
+) if False else None
```

Comment:
Why are you disabling this?

Reply:
I need to run the test on the CI. I will revert once I fix the tests.